package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pensieve",
-  "version": "1.0.12",
+  "version": "1.0.13",
```

Comment:
Can you please revert this? The release pipeline will bump this automatically

Reply:
Done.